app/views/handlers/question.py

```diff
@@ -254,3 +262,44 @@ def clear_radio_answers(self):
                 answer_ids_to_remove, self.current_location.list_item_id
             )
             self.questionnaire_store_updater.save()
+
+    def get_first_incomplete_repeating_block_location(
```

Comment:
Just having a quick look before real review, this one and especially the one below begs for forced named arguments.

Reply:
Yeah good shout

Reply:
Done 👍